pkg/model/model.go

```diff
@@ -1,4 +1,23 @@
 package model
 
+import (
+	"database/sql"
+	"time"
+)
+
 // db.imageを参考にslides structとgenre structを書いて欲しいです(nullとかも反映させてくださると:tasukaru:)
 
+type Slide struct {
+	id          string         `json:"id,omitempty"  db:"id"`
+	dl_url      string         `json:"dl_url,omitempty"  db:"dl_url"`
+	thumb_url   sql.NullString `json:"thumb,omitempty"  db:"thumb"`
```

Comment:
thumb_url に修正して欲しそうです
```suggestion
	thumb_url   sql.NullString `json:"thumb_url,omitempty"  db:"thumb"`
```